src/renderer/inc/IRenderEngine.hpp

```diff
@@ -90,6 +65,8 @@ namespace Microsoft::Console::Render
 
         [[nodiscard]] virtual HRESULT InvalidateTitle(const std::wstring& proposedTitle) noexcept = 0;
 
+        [[nodiscard]] virtual HRESULT PrepareRenderInfo(const RenderFrameInfo& /*info*/) noexcept { return S_FALSE; };

```

Comment:
This is the ONLY non-zero function. Is there a different base file somewhere this def should be within? I swear we have other ones that are defined in the base but are zero in this file.

Reply:
I might be able to put it in `RenderEngineBase`?